ADR/ADR 0006-Markdown.md

```diff
@@ -0,0 +1,35 @@
+# ADR 0006 - Markdown
+
+## Status
+
+ACCEPTED
+
+## Context
+
+After discussion on the best method to implement markdown processing, it was decided to process markdown on the client side interfaces to better facilitate and reduce overheads on the backend API's. In particular, this removes the requirement that we need to transform markdown into HTML except on the client interface, and, when additional client types are added (eg, if a native mobile client is created instead of a web app), they can make direct use of the markdown output and local features without the requirement that we add additional interfaces on the backend application.
```

Comment:
```suggestion
After discussion on the best method to implement markdown processing, it was decided to process markdown on client side interfaces to better facilitate and reduce overhead on the backend API's. In particular, this removes the requirement that we need to transform markdown into HTML except on the client interface, and, when additional client types are added (eg, if a native mobile client is created instead of a web app), they can make direct use of the markdown output and local features without the requirement that we add additional interfaces on the backend application.
```